CHANGELOG.rst

```diff
@@ -7,6 +7,21 @@
 Change log
 ==========
 
+0.7.0 (2023-04-xx)
+------------------
+
+**New features**
```

Comment:
Was it deliberate to put all `0.7` changelog entries here?

Reply:
Yes, I forgot to do it on the PR for v18 I had just merged.